src/main/java/ladder/model/Results.java

```diff
@@ -0,0 +1,27 @@
+package ladder.model;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Results {
+    private List<Result> results;
```

Comment:
```suggestion
    private final List<Result> results;
```

불필요한 재할당을 방지하기 위해 `final` 추가해보는것도 좋을것 같아요!